examples/benchmark_cache.py

```diff
@@ -4,13 +4,8 @@
 import sys
 import time
 import optparse
-import esper
+from esper import component, Processor, World
 
-try:
```

Comment:
Any reason for the removable of matplotlib here?

Reply:
Oh I just took that out to test on this machine which doesn't have matplotlib. 🤣 I'll undo that and rebase.